requirements_test.txt

```diff
@@ -1,47 +1,43 @@
-# linters such as flake8 and pylint should be pinned, as new releases
-# make new things fail. Manually update these pins when pulling in a
-# new version
-
-# types-* that have versions roughly corresponding to the packages they
-# contain hints for available should be kept in sync with them
-
+# Automatically generated by gen_requirements_all.py, do not edit
 -c homeassistant/package_constraints.txt
 -r requirements_test_pre_commit.txt
 astroid==2.12.12
 codecov==2.1.12
 coverage==6.4.4
+fnvhash==0.1.0
 freezegun==1.2.2
 mock-open==1.4.0
 mypy==0.990
+pipdeptree==2.3.1
 pre-commit==2.20.0
 pylint==2.15.5
-pipdeptree==2.3.1
 pytest-aiohttp==0.3.0
 pytest-cov==3.0.0
 pytest-freezegun==0.4.2
 pytest-socket==0.5.1
-pytest-test-groups==1.0.3
 pytest-sugar==0.9.5
+pytest-test-groups==1.0.3
 pytest-timeout==2.1.0
 pytest-xdist==2.5.0
 pytest==7.2.0
 requests_mock==1.10.0
 respx==0.19.2
+sqlalchemy==1.4.42
```

Comment:
This is not a test dependency, but an integration dependency. It doesn't belong here.

Reply:
It is currently required to run any test due to `recorder`'s early import in `conftest`. Please see the PR description/pyproject.toml comment discussion.

Reply:
Unresolving conversation as it is not resolved.

Reply:
Your reply above on the pyproject.toml seemed out of place. 

This requirement is not a test requirement at all, it is an integration requirement and does not belong in this file.